modules/aggregation-layers/src/grid-layer/grid-layer.js

```diff
@@ -89,6 +93,21 @@ export default class GridLayer extends CompositeLayer {
     }
   }
 
+  updateGetValueFuncs(props) {
+    let {getColorValue, getElevationValue} = props;
+    const {colorAggregation, getColorWeight, elevationAggregation, getElevationWeight} = this.props;
+
+    // If a custom `getColorValue` is provided (other than default) use it
+    if (getColorValue === defaultProps.getColorValue.value) {
```

Comment:
If the default value is never used, just set to `null` for clarity.

Reply:
ok, using `null`.